### ISSUE TITLE
Help debugging

### ISSUE THREAD

**Issue by Nudin** [bradand/XMouse#51]
Hi,

I'd like to use this app, but I can't get it working. The ssh connection is set up and running. `xdotool` is installed and if I use connectBot to connect to the host, set DISPLAY as XMouse does and then use xdotool it works. If I use the buttons of XMouse however nothing seems to happen. Is there any way I can debug this? I already tried running sshd with `-dddd` but that does only show the hadshakes, etc but no commands.

**Comment by bradand** [comment]
Hi Michael,

It has been a long time since working on this, it was a for-fun side
project. It sounds like it may be a user issue profile issue or connection
auth issue. The android version used here is very old and the whole thing
needs a modern refresh, no guarantees.

Are you using root user? Can you audit the ssh auth logs to confirm a
connection is established?

**Comment by Nudin** [comment]
No, not using root. ssh-connection is established according to ssh log.

I created a script in `XMouse` with the following content:
```
export DISPLAY=':0' && unset HISTFILE
xdotool type 'foo'
```
Executing this works – however using the type feature of XMouse doesn't.

**Comment by Nudin** [comment]
Update: seems to be an issue with my .bashrc, removing it solves the issue. Probably a bind or alike…

**Comment by Nudin** [comment]
As suspected the issue is due to 'bind's in my .bashrc. Xmouse seems to (falsely?) run bash in interactive mode, therefore the bashrc is exectuded and XMOUSE fails if binds are in .bashrc.

**Comment by Nudin** [comment]
Forget about everything I mentioned so far. After long trial and error it seems that this is non-determistic. Sometimes it works, sometimes not. I'm confused…


Sorry for the many posts here…

**Comment by NitroNils** [comment]
No need to be sorry, reports are good. As an alternative one can edit posts, I presume.
